### PR TITLE
Trials project review

### PR DESCRIPTION
**Pull request by Alhajideen**
This project is one of my project from the microverse trials. I will be reviewing this project myself to see what I could have done better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (Alhajideen) [inline README.md:5]: Please add more description to your readme file. A proper readme will enable one to easily understand the project even before looking through the codes.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    background-color: rgb(133, 127, 127);
+}
+
+
+.small{
+    
+    text-align: center;
+    margin-left: 10px;
+    
+}
```
Comment (Alhajideen) [inline css/style.css:66]: Please remove empty lines within the box.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
+  </body>
+</html>
```
Comment (Alhajideen) [inline index.html:83]: Please use correct html semantic tags and use h1 tag only for major headings. You can use h2-h6 for other smaller headings within the project.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    background-repeat: no-repeat;
+    z-index: -1;
+    background-size: cover;
+    text-align: center;
+    vertical-align: middle;
+    height: 100vh;
+    position: relative;
+    margin-top: 100px;
+}
+#welcome-section h1,#welcome-section p{
```
Comment (Alhajideen) [inline css/style.css:38]: Please add a line after every box. This makes your code neat and easier to understand

**Review by Alhajideen — COMMENTED** [review]
#Changes requested

Thank you for a beautiful work.
Please kindly go through the code and review the few changes request and request another review when done.
All the best and happy coding 😄

File: css/style.css (hunk 2 above)
Comment (Abenezer-Tilahun) [reply css/style.css:66]: - [ ] Please try to includes an explanation, a "why", for each requested change.
Example "Please Why don't you Apply best practices in your HTML code , leaving any empty lines in your HTML code makes your code-base not clear ".

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Alhajideen  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [x] Good Job for phrased the comments politely, as suggestions instead of orders.
- [x] Good Job for using git-hub inline comments to leave review comments.
- [x] Good job for providing Summarize for Requested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**